docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -524,6 +524,21 @@ noted in the location data. Lastly, many location/age/sex groups had fewer than
 50 children, leading to lack of confidence in results. The analysis of this 
 was in `this PR <https://github.com/ihmeuw/vivarium_research_nutrition_optimization/pull/134>`_. 
 
+Similarly, MAM subcategory exposure, the percent of children in "worse" MAM 
+and "better" MAM, will be national only. We analyzed the rate of change 
+between regions in worse MAM fraction and found that it was generally 
+consistent between regions, while pooling for age and sex. 
```

Comment:
Is this pooling among all children under 5 or did we end up excluding the neonates and/or 1-5 month olds?

Reply:
We removed the neonatal age group, but not 1-5 month olds. I'll update the language to include that.